From e3b8d85b3b838e9a5104e795e0b7788879c4775a Mon Sep 17 00:00:00 2001
From: Krzysztof Chojnowski <krzysztof.chojnowski@somlabs.com>
Date: Mon, 22 Dec 2025 16:01:27 +0100
Subject: [PATCH] include only misc-modules and scull subdirs

---
 Makefile | 5 +----
 1 file changed, 1 insertion(+), 4 deletions(-)

diff --git a/Makefile b/Makefile
index 7b08c49..c4f2e2e 100644
--- a/Makefile
+++ b/Makefile
@@ -1,8 +1,5 @@
 
-SUBDIRS =  misc-progs misc-modules \
-           skull scull scullc scullp lddbus sculld scullv shortprint simple tty \
-	   pci usb\
-	   sbull snull short
+SUBDIRS =  misc-modules scull
 
 all: subdirs
 
